doc/Programming-guide/Key-platform-features/Data-collection/Flume-log-appender/index.md

```diff
@@ -0,0 +1,424 @@
+}
+```
+
+Example result:
+
+```json
+{
+    "applicationId": "5",
+    "applicationToken": "82635305199158071549",
+    "confirmDelivery": true,
+    "createdTime": 1466496979110,
+    "createdUsername": "devuser",
+    "description": "Sample Flume log appender",
+    "headerStructure": [
+        "KEYHASH",
+        "VERSION",
+        "TIMESTAMP",
+        "TOKEN",
+        "LSVERSION"
+    ],
+    "id": "131074",
+    "jsonConfiguration": "{\"executorThreadPoolSize\":1,\"callbackThreadPoolSize\":2,\"clientsThreadPoolSize\":2,\"includeClientProfile\":{\"boolean\":false},\"includeServerProfile\":{\"boolean\":false},\"flumeEventFormat\":\"RECORDS_CONTAINER\",\"hostsBalancing\":{\"org.kaaproject.kaa.server.appenders.flume.config.gen.PrioritizedFlumeNodes\":{\"flumeNodes\":[{\"host\":\"10.2.3.93\",\"port\":7070,\"priority\":1},{\"host\":\"10.1.5.67\",\"port\":8070,\"priority\":2},{\"host\":\"10.2.6.44\",\"port\":7090,\"priority\":3}]}}}",
+    "maxLogSchemaVersion": 2147483647,
+    "minLogSchemaVersion": 1,
+    "name": "Sample Flume log appender",
+    "pluginClassName": "org.kaaproject.kaa.server.appenders.flume.appender.FlumeLogAppender",
+    "pluginTypeName": "Flume",
+    "tenantId": "1"
+}
+```
+
+If you want to use Flume agents together with the Flume log appender, create necessary Kaa Flume agents as described in
+[Installing Kaa flume agents](#TODO Add link to Installing Kaa flume agents).
+
+# Setting up Flume log appender
+
+1. As a tenant admin, go to your application >> Log appenders, then click <b>Add log appender</b>.  
+<img src="attach/add-log-appender.jpg">
+2. In the <b>Add log appender</b> window that opens, fill in the required fields  
+In our example, we use <i>Flume</i> as Name.  
+In the <b>Type</b> drop-down list, select Flume.
+Then, specify the fields <b>Flume event format</b> (we selected <i>Records container</i>), and <b>Hosts balancing</b> (we selected <i>prioritized<i>).
+Finally, specify the cluster parameters: host, port and priority. We use localhost:7070. (in our case localhost = 10.2.3.93)  
+<img src="attach/add-log-appender2.png">  
+<img src="attach/add-log-appender3.png">
+3. To finish, click <b>Add</b> at the top of the window.  
+In case of success, you will see your new log appender in the log appenders list.
+
+<img src="attach/add-log-appender4.png">
+
+After that you can go to Data collection demos in Sandbox.
+
+<img src="attach/add-log-appender5.png">
+
+Run the application using the following command in the console:
+
+```bash
+$ java -jar DataCollectionDemo.jar
+```
+
+After this command you will see
+
+<img src="attach/add-log-appender6.png">
+
+This logs you can find in HDFS path which you indicate when set up kaa-flume sink
```

Comment:
Could this be re-written to something like the below?

The logs are stored within the HDFS path defined during Kaa Flume sink setup.